### PR TITLE
Commit google-services.json

### PR DESCRIPTION
**Pull request by martinbonnin**
Make it easier to contribute to the project by adding the firebase API keys. This is ok because the keys are in the APK anyways so there's no absolute way to secure them. They are just a bit more visible now. Also the keys are restricted for the `fr.paug.androidmakers` packageIds and the debug/release SHA1s. 

Because I use the `namespace` API from the AGP, the project now requires Android Studio Dolphin to be able to start the activities from the IDE

### CODE REVIEW

File: app/src/main/java/fr/paug/androidmakers/ui/components/AgendaColumn.kt [hunk 1]
```diff
@@ -79,24 +79,26 @@ fun AgendaColumn(
...
-  ) {
-    Text(
-        modifier = Modifier.padding(horizontal = 12.dp, vertical = 4.dp),
-        text = prettyTime,
-        style = TextStyle(
-            fontFeatureSettings = "smcp"
-        )
-    )
-    Surface(
+  Surface(color = MaterialTheme.colors.background) {
```
Comment (martinbonnin) [inline app/src/main/java/fr/paug/androidmakers/ui/components/AgendaColumn.kt:82]: This is a small unrelated UI change to make sure the time separator is opaque

File: build-logic/src/main/kotlin/fr/androidmakers/gradle/android/application/AndroidApplicationPlugin.kt [hunk 2]
```diff
@@ -22,34 +22,18 @@ class AndroidApplicationPlugin : Plugin<Project> {
...
     addRepositories(project)
-    checkGoogleServices(project)
 
     project.androidSetup()
 
     project.extensions.findByName("android")!!.apply {
       this as BaseExtension
       defaultConfig.apply {
-        applicationId = "fr.paug.androidmakers"
+        namespace = "fr.paug.androidmakers"
```
Comment (martinbonnin) [inline build-logic/src/main/kotlin/fr/androidmakers/gradle/android/application/AndroidApplicationPlugin.kt:31]: I wanted to give a shot at the namespace thing, which took me down the rabbit hole of updating to the latest AS canary because Bumblebee doesn't find the application to start if `package` is not in the Manifest
. It's all working with Dolphin!

File: libs.versions.toml [hunk 3]
```diff
@@ -1,13 +1,13 @@
 [versions]
-kotlin = "1.6.10"
+kotlin = "1.6.10" # bumping to 1.6.20 breaks IDE autocomplete
```
Comment (martinbonnin) [inline libs.versions.toml:2]: OPened https://youtrack.jetbrains.com/issue/KT-52064/1.6.20-break-IDE-autocomplete-in-Android-module-depending-on-mul about this

**Review by BoD — APPROVED** [review]

**Review by oldergod — APPROVED** [review]
